apps/remote_control/lib/lexical/remote_control/project_node.ex

```diff
@@ -0,0 +1,188 @@
+defmodule Lexical.RemoteControl.ProjectNode do
+  alias Lexical.Project
+  alias Lexical.RemoteControl
+  require Logger
+
+  defmodule State do
+    defstruct [
+      :project,
+      :node,
+      :paths,
+      :cookie,
+      :stopped_by,
+      :stop_timeout,
+      :started_by,
+      :status
+    ]
+
+    def new(%Project{} = project) do
+      cookie = Node.get_cookie()
+
+      %__MODULE__{
+        project: project,
+        node: node_name(project),
```

Comment:
why do we need this field? it seems like it's easy to calculate given the project.